package.json

```diff
@@ -121,5 +122,9 @@
   },
   "optionalDependencies": {
     "fsevents": "*"
+  },
+  "engines": {
+    "node": "^6",
```

Comment:
Does this prevent 7?

Reply:
yes the caret

> Allows changes that do not modify the left-most non-zero digit in the [major, minor, patch] tuple.

https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

however i'll update it to `6.x` because it's a simpler syntax